GVFS/GVFS.Common/InternalVerbParameters.cs

```diff
@@ -4,14 +4,16 @@ namespace GVFS.Common
 {
     public class InternalVerbParameters
     {
-        public InternalVerbParameters(string serviceName, bool startedByService)
+        public InternalVerbParameters(string serviceName, bool startedByService, string maintenanceJob)

```

Comment:
It was unclear to me from just reading the code what a `maintenanceJob` string is....

Reply:
I think once you know what a mintenance job is it makes sense.  Let me know if comments would be helpful or what other suggestions you have for clarity.

Reply:
Could this be an `enum` so that it is well defined?

Reply:
I prefer to keep it a string at least for now.  Since it is used mostly for calling from function tests I think it's clearer to send in "LooseObjects" then say "0".  Let me know if I'm missing something.

Reply:
I was hoping that you could still pass in "LooseObjects" and the json deserialize would parse it into the enum correctly.  If not there are some checks in place so it should be okay.